sonar-scala-plugin/src/test/java/org/sonarsource/scala/plugin/ScoverageSensorTest.java

```diff
@@ -121,9 +121,9 @@ public void testLogInvalidXMLFile() throws IOException {
 
     new ScoverageSensor().execute(context);
 
-    String expectedMessage = String.format("File '" + reportPath.toString() + "' can't be read. javax.xml.stream.XMLStreamException: ParseError at [row,col]:[2,1]\nMessage: Content is not allowed in prolog.");
```

Comment:
Why did we lose a part of the message?

Reply:
The build was failing on my windows, because the end of the message was a little bit different with my JVM, and I needed this change so I could test my sensor on Windows.